miniapps/common/makefile

```diff
@@ -37,15 +38,18 @@ endif
 %.o: %.cpp
 
 # Replace the default implicit rule for *.cpp files
-%.o: $(SRC)%.cpp $(CONFIG_MK)
-	$(MFEM_CXX) $(MFEM_FLAGS) -c $(<) -o $(@)
+%.o: $(SRC)%.cpp $(MFEM_LIB_FILE) $(CONFIG_MK)
+	$(MFEM_CXX) $(MFEM_FLAGS) -c $(<) -o $(@) $(MFEM_LIBS)
 
```

Comment:
The addition of `$(MFEM_LIBS)` seems unnecessary and it is causing warnings in the regression tests and travis-ci Mac builds. The added dependency on `${MFEM_LIB_FILE}` also seems unnecessary.

Reply:
Can you fix plz?

Reply:
I'm not sure where those came from but I'd be happy to fix them.